src/commands/PublishCommand.js

```diff
@@ -362,7 +362,7 @@ export default class PublishCommand extends Command {
       const version = this.updatesVersions[depName];
 
       if (deps[depName] && version) {
-        deps[depName] = this.flags.exact ? version : "^" + version;
+        deps[depName] = this.getOptions().exact ? version : ("^" + version);
```

Comment:
I think we should cache this `getOptions()` result outside of the `forEach`, since it will never change. I appreciate the disambiguating parentheses on the concatenation ternary.